configs/projects/puppet-agent.rb

```diff
@@ -241,7 +241,12 @@
   end
   proj.component "ruby-shadow" unless platform.is_aix? || platform.is_windows?
   proj.component "ruby-augeas" unless platform.is_windows?
-  proj.component "openssl"
+
+  fips_build = ENV["USE_SYSTEM_OPENSSL"]
+  if fips_build != '1'
+    proj.component "openssl"
+  end
```

Comment:
I believe you can add a `do not merge` label to this PR. You can also add "(WIP)" to the PR title.